js/ui/appDisplay.js

```diff
@@ -2671,9 +2671,10 @@ class AppCenterIcon extends AppIcon {
         let app = appSys.lookup_app(EOS_APP_CENTER_ID);
 
         super._init(app, viewIconParams, iconParams);
+    }
 
-        this._id = EOS_APP_CENTER_ID;
-        this._name = this.app.get_generic_name();
+    get name() {
+        return _("More Apps");
```

Comment:
Doing this this way means that someone (hi) will need to go and copy over all the translations from gnome-software, so I'm prepared to be talked out of it, but I think the decision to show “More Apps” is more a property of our shell than it is of our app center package.

Reply:
Alternatively, you can just move the call to `super._init()` to below these lines